xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp

```diff
@@ -45,6 +46,9 @@ bool CWinSystemGbmGLESContext::InitWindowSystem()
   CLinuxRendererGLES::Register();
   RETRO::CRPProcessInfoGbm::Register();
   RETRO::CRPProcessInfoGbm::RegisterRendererFactory(new RETRO::CRendererFactoryOpenGLES);
+#if defined(HAS_GBM_BO_MAP)
```

Comment:
this detail can be buried in `RegisterRendererFactory()` (and possibly deeper)

Reply:
Not sure how that would benefit? Here it is in a gbm specific place anyways.